python/cudf/cudf/utils/ioutils.py

```diff
@@ -1238,6 +1307,13 @@
     compressed file (upper 48 bits).
     The start offset points to the first byte to be read, the end offset points
     one past the last byte to be read.
+storage_options : dict, optional, default None
+    Extra options that make sense for a particular storage connection,
+    e.g. host, port, username, password, etc. For HTTP(S) URLs the key-value
+    pairs are forwarded to ``urllib.request.Request`` as header options.
+    For other URLs (e.g. starting with “s3://”, and “gcs://”) the key-value
```

Comment:
I keep thinking I'm done fixing smart quotes... I won't add any more suggestions for this. Please apply to any places I missed. 😅 
```suggestion
    For other URLs (e.g. starting with "s3://", and "gcs://") the key-value
```